readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Zaransoftline.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Zaransoftline.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to PR description
<img width="925" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/6729a475-74dc-4d90-9282-0c0df801e18b">